### ISSUE TITLE
FortiusAnt with Tacx trainer software

### ISSUE THREAD

**Issue by novigado** [WouterJD/FortiusANT#383]
This is probably not an issue with fortiusAnt, but I will start here and close the issue if that is the conclusion.

I have an old i-flow trainer, which I got to work with Zwift and RGT on both pc and mac. Excellent stuff WouterJD!

I also got it to work with the Tacx Trainer Software Android app, and with the windows desktop app (only that if thinks it has found a Tacx Neo 2t...)

But, I'm unable to get it to work with the desktop Tacx app for Mac. "No trainer found" is the only information I have. 

Any ideas/pointers on what could be causing this? Very little info, I know, sorry.

**Comment by WouterJD** [comment]
A. Tacx Neo is found, because there is no registered ID for a FortiusAnt smart trainer

B. [assumption] TTS on PC most likely does not discover tacx-specific info and hence knows it's not a Tacx; tacx tried to keep the environment closed. If anybody has a suggestion how to resolve, I'd be glad to implement

**Comment by novigado** [comment]
Wow, that's quick! Thanks!

I don't know the code, or much about how the signalling looks, but I'm guessing that each device is sending out an identifier during some kind of handshake? If that is the case, then maybe it would be possible to "steal"/piggyback on another devices ID? Maybe a device that is sufficiently similar to the one you are pedaling on?

**Comment by WouterJD** [comment]
As said, suggestions welcome.
If you could tell me the protocol, I will implement.

The fact that all Cycling Training Programs, including TTS/Android can pair is enough hint that TTS/Mac is an odd duck in the pool

**Comment by novigado** [comment]
OK, you think the mac app is actually using a different protocol altogether? I think I will have to have a look at the code...

The tacx android app, by the way, identifies fortiusant as "Blue Motion"

**Comment by WouterJD** [comment]
Welcome to the FortiusANT community, @novigado 
- - -
I'm always curious to know who I communicate with, where FortiusANT is used and what configuration is used.
Please tell me what bundle did you buy, and what brake and what head unit do you use?
I would therefore appreciate that you introduce yourself; perhaps leave a comment under issue #14.
You could also consider to visit [the sponsor page](https://github.com/sponsors/WouterJD)
- - -

**Comment by novigado** [comment]
Well, yes of course. Hello!

My name is Peter and I live in Göteborg, Sweden. I have an I-Flow trainer with an old Cube racer sitting on top. I think this means I have a 1901 break. The head unit is a T1932. Just purchased it second hand for almost nothing, thinking it had bluetooth support. When I realized it didn't I bought another trainer, a Tacx Flow Smarttrainer, but that one was so much noisier than the I-flow so I sold it again, keeping only the ANT+ dongle. The reason I kept it was because I had stumbled upon your software, and so here we are.

I have installed FortiusAnt on my Macbook Pro 2014 (and on my PC) and I run Zwift and RGT Cycling, though Zwift is on trial which is soon to expire...

For work I do software development. Or not so much actual development anymore, guess I'm too old to keep up with the youngsters. I know how to write a PR at least :-)
